build.gradle

```diff
@@ -2,6 +2,9 @@ apply plugin: 'groovy'
 
 repositories {
   mavenCentral()
+  maven {
```

Comment:
I don't understand why this is required for the version bump?

Reply:
JenkinsPipelineUnit is not being published to Maven Central anymore, just to the Jenkins repository. See https://github.com/jenkinsci/JenkinsPipelineUnit/issues/173 if you are really curious.